mlos_core/mlos_core/tests/optimizers/optimizer_test.py

```diff
@@ -203,7 +195,17 @@ def objective(point: pd.DataFrame) -> pd.Series:   # pylint: disable=invalid-nam
         space_adapter_type=SpaceAdapterType.LLAMATUNE,
         space_adapter_kwargs=space_adapter_kwargs,
     )
+
```

Comment:
```suggestion
    # pylint: disable=too-many-locals
```
(should be at the beginning of the function, actually)